packages/element/src/index.js

```diff
@@ -45,6 +46,24 @@ import serialize from './serialize';
  */
 export { createElement };
 
+/**
+ * Creates a simplified element representation.
```

Comment:
_What is_ a simplified element, and why would I need one?

This is my main concern with this change, mostly that we have this separate concept of an element which is very incidental to a specific need for blocks, though could be a reasonable replacement for element altogether. So I don't think it's strictly unsuitable for `@wordpress/element`, though its existence parallel to the "full" element is unfortunate. The previous implementation where it existed in `blocks` was more accurate to the fact that it's an implementation detail therein, but then is complicated by the serializer's awareness of this shape.

I don't really have an answer here, just expressing thoughts in my head in response to your desire to move forward ( https://github.com/WordPress/gutenberg/pull/7476#issuecomment-400593698 ).

Couple options to consider:

- `createElement` should return the simplified form, and we somehow teach React to deal with it in rendering.
- We move this back to `blocks`, but with some adapter which prepares an element for serialization, even if right now that just means returning the element verbatim.
   - Speaks also to the fact that we're misaligned between what React's renderer expects and our custom serializer can handle

Reply:
What if we just consider this as a separate thing? It's just the HTML format of Gutenberg. Doesn't need to be coupled with element at all. Though I don't know what name to give it.